packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -451,10 +449,9 @@ Function CWOCConfig() {
         Copy-Item "${PREDEFINED_CONFIG_DATA}" -Destination "${YAML_DIR}/default.tmp"
         Write-Output "Successfully fetched the config and saved in ${YAML_DIR}\default.tmp"
     } else {
-        & cmd /c "`"$CWAProgramFiles\config-downloader.exe`" --output-dir `"${YAML_DIR}`" --download-source `"${OtelConfigLocation}`" --mode ${param_mode} --config `"${COMMON_CONIG}`" --multi-config ${multi_config} 2>&1"
-        # Use return instead of exit since CWAgent and ADOT Collector should be two independent agents as much as possible
+        & $CWAProgramFiles\config-downloader.exe --output-dir "${YAML_DIR}" --download-source "${OtelConfigLocation}" --mode "${param_mode}" --config "${COMMON_CONIG}" --multi-config "${multi_config}"
         if ($LASTEXITCODE -ne 0) {
-           return
+            return

```

Comment:
nit seems an extra empty space.

Reply:
was probably from the revert. I'll fix it

Reply:
So, I counted the spaces, and it actually lines up correctly with this change. the indent is 4 with this PR but was 3 before 🤷